tests/test_transformer.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import annotations
+
+import sys
+from enum import Enum
+from types import FunctionType  # noqa: F401
+from typing import Dict, Tuple, cast
+from unittest import skipIf
+
+from basedtyping import ForwardRef, Intersection
+from basedtyping.transformer import eval_type_based
+from typing_extensions import Annotated, Callable, Literal, TypeGuard, TypeIs, Union
+
+# ruff: noqa: PYI030, PYI030
```

Comment:
```suggestion
# ruff: noqa: PYI030
```
though this should be removed completely if u do my suggestion to change it to `Literal[1, 2]`. if not maybe add a comment explaining why

Reply:
this is resolved